recipes/evolutionary_keras/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "evolutionary_keras" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 
+  url: https://github.com/N3PDF/evolutionary_keras/archive/v{{version}}.tar.gz
+  sha256: 39dc8ff742cea271b67605c029fb0ae66bbfa1cc2dd5574ee372d0ed81234a83
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - keras
+    - numpy
+
+test:
+  imports:
+    - evolutionary_keras
+  requires:
+    - pytest
+  commands:
+    - pytest --pyargs evolutionary_keras
+
+about:
+  home: https://github.com/N3PDF/evolutionary_keras
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE
+  summary: "evolution_keras implements Genetic Algorithms for Keras"
+
+
+extra:
+  recipe-maintainers:
+    - scarlehoff
+    - scarrazza
+    - roystegeman
```

Comment:
Are you willing to be maintainers

- @scarrazza
- @roystegeman

Reply:
Yes, I am willing to be a maintainer.

Reply:
@scopatz , yes.